### DISCUSSION TITLE
After update of AMD drivers and fancontrol, GPU temps are gone

### DISCUSSION THREAD

**Discussion by NorthLight-EWR** [Rem0o/FanControl.Releases#2813] [General]
The GPU and hotspot temp is gone, not availlable any more.
In Adrenalin, I can still see the temps working.

So what can I do to get the GPU temps back?

I have already:

- Downgraded to last Adrenalin edition with previous driver version. No change
- Downgraded to previously working fancontrol. No change
- Completely uninstalled both, fancontrol and AMD drivers and AMD adrenalin. Rebooted, and reinstalled both with actual version. No change

**Comment by Rem0o** [comment]
When you update Adrenaline, always check the "clean install/full install" option.  Use DDU before if you want to be extra safe.  I've had bad luck doing the non-full update. 

Otherwise, do you got ADLX enabled and any logs?

**Comment by NorthLight-EWR** [comment]
Yes, did a full uninstall and install and then a full install.
What  does "enable ADLX" mean?

**Reply by Rem0o** [reply]
In sensor source, settings page, thats the amd driver.

**Comment by NorthLight-EWR** [comment]
yep, it was and is:
![image](https://github.com/user-attachments/assets/da6ae7b7-f44d-403f-9ad5-08902714ad26)

**Reply by NorthLight-EWR** [reply]
This is all, I can get. Both RX6800 GraKa sensors are missing now..
![image](https://github.com/user-attachments/assets/97079e04-8087-44bb-a332-1bd9ec81a803)

**Comment by Rem0o** [comment]
Any log.txt?

**Comment by NorthLight-EWR** [comment]
Unfortunately only old entries, nothing actual. I did reload sensors, but didn't generate even one message in the log with todays' datetime.

**Reply by Rem0o** [reply]
Which version of adrenaline you got?

**Reply by NorthLight-EWR** [reply]
> Which version of adrenaline you got?

Got v23.12.1, so not the latest, because of "Virtual Desktop". But I had, as mentioned above, the latest also installed and did not succed.

**Comment by stupdude2** [comment]
I updated AMD drivers and lost GPU temps. Fixed by using the AMD Cleanup Utility and reinstalling drivers: https://www.amd.com/en/resources/support-articles/faqs/GPU-601.html

**Reply by NorthLight-EWR** [reply]
Ok, will give it a try now

**Reply by NorthLight-EWR** [reply]
Well. Worked. So uninstalling adrenalin does not completely uninstall it. Not fine.
Thanks for your support :)

Solved by adapting to bugs of AMD.
